source/SIL.AppBuilder.Portal.Frontend/src/ui/components/project-table/project-table.scss

```diff
@@ -1,7 +1,82 @@
-.project-actions {
-  color: #4a4a4a;
+.project-table {
 
-  i {
-    opacity: 0.5;
+  color: #515151;
+  font-family: Montserrat;
+  font-size: 13px;
+
+  a {
+    font-family: Montserrat;
```

Comment:
font family and size _shouldn't_ (?) need to be specified everywhere

Reply:
Yeah, not sure why but doesn't get inherit from body as they should

Reply:
what's overriding it?